docs/tutorials/08_quantum_kernel_trainer.ipynb

```diff
@@ -240,13 +1238,22 @@
    "execution_count": 4,
    "id": "a190efef",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/tmp/ipykernel_100988/4072072807.py:5: DeprecationWarning: __init__: the training_parameters argument is deprecated as of version 0.4.1 and will be removed no sooner than 3 months after the release. Instead use the training_parameters argument.\n",
+      "  quant_kernel = QuantumKernel(fm, training_parameters=training_params, quantum_instance=backend)\n"
+     ]
```

Comment:
Steve, the notebook seems corrupted.  I made what should have been innocuous changes manually, and yet somehow a *ton* of changes have appeared, and I am a little mystified as to how that happened, b/c I certainly didn't make these changes.  I'm reverting and re-applying my minimal changes.

Reply:
Some changes we might expect when re-running the notebook as it will re-generate the output cells, and if thats an image it may well end up with a huge diff. Also, depending on how the notebook was created and what version you are running it may well end up adding tags to the cell, or removing them. Generally I just look for difference that seems to be outside of what I might expect.